code/modules/clothing/suits/xeno/tajara.dm

```diff
@@ -0,0 +1,168 @@
+/obj/item/clothing/suit/storage/toggle/labcoat/tajaran
+	name = "people's republic medical coat"
+	desc = "A sterile insulated coat made of leather stitched over fur."
+	icon = 'icons/obj/tajara_items.dmi'
+	icon_state = "medcoat"
+	item_state = "medcoat"
+	icon_open = "medcoat_open"
+	icon_closed = "medcoat"
+	contained_sprite = TRUE
+	description_fluff = "Today the fashion that dominates Adhomai shares few similarities to the clothing of old Furs, linen, hemp, silk and other such fabrics were traded for \
+	synthetic versions, creating a massive boom in the nylon industry and textile industry in the cities. Jeans, overcoats, army uniforms, parade uniforms, flags, pants, shirts, ties, \
+	suspenders, overalls are now the fashion of every Tajara from Nal'Tor to Kaltir. The protests of \"Old fashion\" supporters usually shatter against how effective and cheap it is to \
```

Comment:
'The protests of "Old Fashion" supporters usually can't stand against how undeniably effective and cheap to produce Human clothes are.' -- same for all other items sharing this description